common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -206,6 +206,16 @@ trait CommercialSwitches {
     exposeClientSide = false
   )
 
+  val MembershipEngagementBanner = Switch(
+    SwitchGroup.Commercial,
```

Comment:
I just chatted with @SiAdcock and @gtrufitt. Personally I think it makes sense for membership to have their own switch group as it is one of the core things we are doing now. However, it is completely up to the membership team to decide this and implement it. 

Certainly I think it's outside the scope of this PR.